services/updatePsyFields.ts

```diff
@@ -10,7 +12,7 @@ const addFrenchLanguageIfMissing = (languages) => {
   return languages;
 };
 
-const editablePsyFields = (psy) => ({
+export const editablePsyFields = (psy: Psychologist): EditablePsychologist => ({
```

Comment:
All the functions here are exported twice (here and at the end of the file)

Reply:
I think it's ok for now, we will need to review all of our export in this task => https://trello.com/c/NYIiP3Dk/154-have-propre-export